_includes/assets/sass/_components.scss

```diff
@@ -0,0 +1,23 @@
+/* ==========================================================================
+COMPONENTS
+========================================================================== */
+
+// Title bar
+.title-bar {
+	background: $gradient;
```

Comment:
Not sure about browser support requirements but do we need a fall back flat colour for this?

Reply:
I think you are right @DarkSnow , I've added a fallback.